repos/iiif-storage/src/Controller/ManifestController.php

```diff
@@ -80,13 +81,14 @@ public function __construct(
         $this->apiRouter = $apiRouter;
     }
 
-    public function addCollectionToViewModel(array $vm, string $collectionId, string $manifestId) {
+    public function addCollectionToViewModel(array $vm, string $collectionId, string $manifestId)
+    {
         if ($collectionId) {
-            $collection = $this->collectionRepo->getById($collectionId);
-            if (!$this->collectionRepo->containsManifest($collection, $manifestId)) {
+            if (!$this->collectionRepo->containsManifest((int)$collectionId, $manifestId)) {
                 throw new NotFoundException();
             }
-            $collectionRepresentation = $this->collectionBuilder->buildResource($collection, $this->shouldUseOriginalIds());
+            $collection = $this->collectionRepo->getById($collectionId);
+            $collectionRepresentation = $this->collectionBuilder->buildResource($collection, $this->shouldUseOriginalIds(), 1, 1, 0);
```

Comment:
1 - manifest
1 - canvas
0 - offset

Reply:
I can't make sense of what's going on here. What's `buildResource()` do?

Reply:
So the various `buildResource` methods take an Omeka `ItemSetRepresentation` or `ItemRepresentation` and return a [wrapped representation](https://github.com/digirati-co-uk/madoc-platform/blob/feature/performance/repos/iiif-storage/src/Model/ManifestRepresentation.php) that gives access to:
- Original Omeka item and Omeka ID
- Underlying IIIF resource through IIIF-PHP
- Pagination if its a partial resource

It's a cleaner alternative to the `addMetadata` approach on IIIF-PHP objects used before to store things like the Omeka idea. This object is intended to be passed around and will probably grow as more data is built into manifests (such as statistics). 

Maybe the build resource should move to a factory. Something like:
```php
... ->buildResource(
    (new CollectionResource::fromItemSet($collection))
          ->limitManifestsLoaded(1)
          ->limitCanvasesLoaded(1)
          ->canvasOffset(0)
          ->useOriginalIds()
);
```